poseidon/Cargo.toml

```diff
@@ -21,14 +21,18 @@ ark-sponge = { git = "https://github.com/penumbra-zone/sponge", branch = "r1cs",
 poseidon-parameters = { git = "https://github.com/penumbra-zone/poseidon377", rev = "50699746", default-features = false }
 poseidon-permutation = { git = "https://github.com/penumbra-zone/poseidon377", rev = "50699746", default-features = false }
 
-[build-dependencies]
-ark-bls12-381 = { version = "^0.3.0" }
-ark-ff = { version = "^0.3.0" }
+poseidon-paramgen = { git = "https://github.com/penumbra-zone/poseidon377", rev = "50699746" , optional = true, default-features = false  }
+
+[lib]
+name = "liminal_ark_poseidon"
 
-poseidon-paramgen = { git = "https://github.com/penumbra-zone/poseidon377", rev = "50699746" }
+[[bin]]
+name = "liminal-ark-poseidon-paramgen"
+path = "src/generate_parameters.rs"
+required-features = ["std", "build-binary"]
 
 [features]
-default = ["std"]
+default = ["build-binary"]
```

Comment:
why `build-binary` is default? and why `std` is not default anymore?

Reply:
It's default to not to have to use it in build and `std` is not to make `cargo nono` happy.